### PR TITLE
fix aptly logic

### PR DESCRIPTION
**Pull request by mburns**
* improve test coverage

### CODE REVIEW

**Comment by drenalin23** [comment]
+1 after fixing the rubocop issues that are causing circleCI to fail

**Comment by mburns** [comment]
This is currently failing because the `ele-apt::repo` recipe isn't included (correctly) in the run_list, thus, tests against the `::repo` recipe are failing.

I tried editting the `.kitchen.cloud.yml` to have a "default" profile and a "repo" profile, to test each recipe separately, but must have done something wrong...

**Comment by drenalin23** [comment]
To fix the default_spec.rb change create_apt_repository to add_apt_repository.  That allows that unit test to pass.

File: test/unit/spec/repo_spec.rb [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
   let(:chef_run) { ChefSpec::SoloRunner.new(step_into: ['aptly']).converge(described_recipe) }
 
+  it 'includes the `aptly::install` recipe' do
+    expect(chef_run).to include_recipe('aptly::install')
+  end
+
   it 'creates a directory with an explicit action' do
-    expect(chef_run).to create_directory('/data/aptly')
+    expect(chef_run).to create_directory('/opt/aptly')
   end
```
Comment (drenalin23) [inline test/unit/spec/repo_spec.rb]: You could also add
  it 'creates a directory with an explicit action' do
    expect(chef_run).to create_directory('/data/packages')
  end

to increase the coverage on this unit test to 100%.

**Comment by drenalin23** [comment]
Tracking this down - trying to get kitchen verify to work on my local workstation.  So aptly::install doesn't work because we are trying to override the distribution name 'squeeze' as 'trusty'.  The repo.aptly.info site does not have a trusty tree - only squeeze.  Note at the bottom of this main page (http://repo.aptly.info/) it says to not worry about using the squeeze install.   I'm going to try taking the override out and see if I can get kitchen verify running (and aptly installed).

**Comment by drenalin23** [comment]
If we comment out 
# node.set['aptly']['dist'] = 'trusty'
Then aptly will install correctly on a kitchen verify, but then we die on the aptly_repo section.  I'll check in on this in the morning with mburns.

File: recipes/repo.rb [hunk 2]
```diff
@@ -7,16 +7,26 @@
...
+packagedir = '/data/packages'
+
+directory packagedir do
   recursive true
+  owner node['aptly']['user']
+  group node['aptly']['group']
 end
 
 aptly_repo 'aptly' do
   action :add
```
Comment (drenalin23) [inline recipes/repo.rb]: I think change this to action :create

**Pull request merged**